controllers/boardsController.js

```diff
@@ -0,0 +1,103 @@
+const Logger = require("../logger/Logger");
+const logger = new Logger();
+const BoardsRepository = require("../repositories/boardsrepository");
+const boardsRepository = new BoardsRepository();
+
+
+
+const Error = (msg) => {
+    logger.log(msg);
+}
+
+exports.boardsController = {
+
+    getBoards(req, res) {
+        logger.log("Getting all boards");
+        res.status(200).json(boardsRepository.getAllBoards());
+    },
+
+    getBoard(req, res) {     
+        const result = boardsRepository.searchBoard(req.params.id);
+        if (result == null) {
+            const msg = 'Error! boarder no ' + req.params.id + ' was not found!';
+            Error(msg);
+            res.status(400).json(msg);
+        }
+        else {
+            logger.log("get board");
+            res.status(200).json(result);
+        }  
+    },
+
+    updateBord(req, res) {
+        const ifExits = boardsRepository.searchBoard(req.body.id);
+        if(ifExits == null) {
```

Comment:
this if statement doesn't make sense. 
also, use ES6 compare type check 
improve readability.